src/client/clientmap.cpp

```diff
@@ -309,11 +317,10 @@ void ClientMap::renderMap(video::IVideoDriver* driver, s32 pass)
 		if (!block->mesh)
 			continue;
 
-		float d = 0.0;
-		if (!isBlockInSight(block->getPos(), camera_position,
-				camera_direction, camera_fov, 100000 * BS, &d))
```

Comment:
Also, removal of this makes `camera_direction` and `camera_fov` dead variables.

Reply:
Done